tests/api/route-services/pipeline-response.test.js

```diff
@@ -0,0 +1,127 @@
+const AWSMock = require('aws-sdk-mock');
+const ioClient = require('socket.io-client');
```

Comment:
Is it possible to use https://github.com/supremetechnopriest/socket.io-mock#readme for these tests? We are using it already in https://github.com/biomage-ltd/api/blob/master/tests/utils/cache-request.test.js and https://github.com/biomage-ltd/api/blob/master/tests/api/event-services/work-request.test.js

Reply:
I tried using `MocketSocket`, but it's failing on caling `io.sockets`. I think there are some interface differences between the mock library which mocks a socket, and its usage  in `pipeline-response` which gets available sockets in the SocketIO Express middleware.